modules/analysis-runner/src/cljdoc/analysis/runner_ng.clj

```diff
@@ -9,26 +9,21 @@
             [clojure.edn :as edn]
             [cljdoc.util :as util]
             [cljdoc.analysis.deps :as deps]
-            [cljdoc.analysis.runner :as runner]
+            [cljdoc.analysis.runner :as runner :refer [copy]]
```

Comment:
Not a big fan of using `:refer` in general because it hides the origin of a function. Explicit `:refer` is much better than `:refer :all` in that regard but IMHO it's best to just avoid `:refer` whenever possible.

Reply:
resolved via moving `copy` to `util`